README.md

```diff
@@ -1,7 +1,9 @@
 mupen64plus-input-bot
 =====================
 
-An input driver for [mupen64plus](https://github.com/mupen64plus/mupen64plus-core) that consumes a JSON response from a web server. Intended for use with my [TensorKart](https://github.com/kevinhughes27/TensorKart) project that trains an AI using TensorFlow to play the classic N64 title MarioKart 64.
+An input driver for [mupen64plus](https://github.com/mupen64plus/mupen64plus-core) that consumes a JSON response from a
+web server. Intended for use with my [TensorKart](https://github.com/kevinhughes27/TensorKart) project that trains an AI
+using TensorFlow to play the classic N64 title MarioKart 64.
```

Comment:
I usally just let the text editor wrap the lines, why did you want to do it manually?

Reply:
Just preference. Some editors don't wrap, and showing a diff can be difficult to compare in that case since you get half your screen width.

I'd be happy to revert this back if you prefer to leave it one line.

Reply:
(honestly, that's actually why I did it in a separate commit)

Reply:
pros and cons I guess. Most my readme's rely on proper wrapping from the text editor / github display. For consistency I would appreciate the revert :)

Reply:
Sure, I'll take care of it as soon as I get back to this. Hopefully in the next couple days, but we'll see.